server/cloudformation/ecs.go

```diff
@@ -32,19 +32,25 @@ type LoadBalancer struct {
 type ECSServiceProperties struct {
 	ServiceName    *string
 	Cluster        *string
-	DesiredCount   *IntValue
-	LoadBalancers  []LoadBalancer
+	DesiredCount   *IntValue      `hash:"ignore"`
+	LoadBalancers  []LoadBalancer `hash:"set"`
```

Comment:
This ensures that the ordering of the load balancers doesn't cause a replacement.